Added CumulusCI Config

jlantz opened the pull request:
<!--
Thank you for submitting a pull request to SchemaStore.

Before continuing, please read the guidelines:
https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md

Adding a JSON schema file to the catalog is required.
Add tests files. (.json, .yml, .yaml or .toml)
Use the most recent JSON Schema version that's well supported by editors and IDEs, currently draft-07.
JSON formatted according to the .editorconfig settings.
-->

This is my first contribution. I've read the CONTRIBUTING.md doc and believe I've followed the instructions for adding a remotely hosted json schema to the repo. This PR adds the schema for CumulusCI's configuration file. CumulusCI is an open source automation framework for Salesforce projects configured through a cumulusci.yml file. The project maintains its json schema directly in the repo.

jlantz commented:
This is failing on prettier issues in 3 files, `catalog.json`, `test/prettierrc/.prettierrc.yml`, and `test/prettierrc/prettierrc.json`. I didn't modify the test/prettierrc/* files in this PR.

It appears prettier is trying to make broad formatting changes to the catalog.json that seem unwise for readability...
```
diff --git a/src/api/json/catalog.json b/src/api/json/catalog.json
index 5b1174da..87cbcff1 100644
--- a/src/api/json/catalog.json
+++ b/src/api/json/catalog.json
@@ -16,29 +16,19 @@
     {
       "name": "1Password SSH Agent Config",
       "description": "Configuration file for the 1Password SSH agent",
-      "fileMatch": [
-        "**/1password/ssh/agent.toml"
-      ],
+      "fileMatch": ["**/1password/ssh/agent.toml"],
       "url": "https://developer.1password.com/schema/ssh-agent-config.json"
     },
     {
       "name": "Application Accelerator",
       "description": "Application Accelerator for VMware Tanzu",
-      "fileMatch": [
-        "accelerator.yaml"
```

hyperupcall commented:
Hmm that's odd with the - `test/prettierrc` files. Prettier usually modifies them (via pre-commit) locally on my machine, but they are usually not modified in CI of PRs.

Do you think you can revert the "Fix JSON syntax" commit and push? pre-commit shouldn't complain or do any formatting in CI if there is like a +4 line addition in `catalog.json`

jlantz commented:
OK, looks like VS Code helpfully reformatted the entire file. I reverted and fixed the json syntax issue. Looks like it's good to merge now. Thanks for the help.

hyperupcall commented:
Thank you :)
